packages/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,285 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Tab, { TabProps } from './Tab';
+import TabTitle from './TabTitle';
+
+const listStyle = css`
+  display: flex;
```

Comment:
Looking at the flex properties on this, it looks a lot like this doesn't need to use flexbox.

Reply:
you're right 👍